src/data/list/perm.lean

```diff
@@ -891,6 +891,14 @@ suffices ∀ {l₁ l₂}, l₁ ~ l₂ → pairwise R l₁ → pairwise R l₂, f
     exact h _ (p'.symm.subset m) }
 end
 
+lemma pairwise.perm {R : α → α → Prop} {l l' : list α} (hR : l.pairwise R)
```

Comment:
```suggestion
lemma pairwise.congr_perm {R : α → α → Prop} {l l' : list α} (hR : l.pairwise R)
```
Assuming we have names of this form elsewhere

Reply:
We don't have such names elsewhere